cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -33,6 +34,23 @@ else()
   set(configure_command "./config")
 endif()
 
+if(ANDROID)
+  if(NOT "${CMAKE_ANDROID_ARCH}" STREQUAL "")
```

Comment:
Minimum CMake version for Hunter is 3.0 so we can hit CMP0054 policy problem, please use `string(COMPARE EQUAL ...)` alternative: http://0.readthedocs.io/en/latest/cmake.html#strequal